src/components/Accordion/Accordion/__tests__/accordion.interactions.ts

```diff
@@ -28,6 +28,7 @@ async function openAndCheckAccordionItem(canvas: Screen, title: string) {
   const before = getOpenedAccordionItem(canvas);
   const elHeading = getAccordionHeadingByText(canvas, title);
   userEvent.click(elHeading);
+  await delay(CHANGES_DELAY);
```

Comment:
we prefer actively waiting for something to happen (polling) and not "sleeping".

Reply:
I don't mind to try it out, just tell me how it should look, cause I found only this pattern in our interactions tests

Reply:
waitFor should work?

Reply:
That's what you've meant @shlomitc ?